dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -273,6 +273,33 @@ public void TipFollowsTargetOnWindowResize()
 
                     Verify.IsLessThan(GetTipVerticalOffset(), initialTipVerticalOffset);
                 }
```

Comment:
I am not quite sure, if we still need this test. According to the test name, we wanted to test for the window size change, however this test did not fail, even despite the window changing size and the TeachingTip not updating correctly in every case.

Reply:
I think that the code that the test app uses to change the window size does not exercise the same code path as clicking on the maximize button. So we didn't have coverage of this exact scenario.

Reply:
Interesting. However I was able to reproduce the issue this PR fixes with the windows key + up/down arrow. Anyway, the test now correctly checks for #1547.